ansible/image.yml

```diff
@@ -0,0 +1,19 @@
+---
+- import_playbook: setup.yml
```

Comment:
Maybe the [SSH keys setup](https://github.com/aeternity/infrastructure/blob/6f100077951b7f73384f0a90a6eeb63708c5d145/ansible/setup.yml#L9-L23) shall be moved out of the AMI creation?

Reply:
What's the reason?

This is one of the slowest tasks ATM.

Reply:
Ok. Please refer to https://github.com/aeternity/infrastructure/pull/61#discussion_r194372872